meta-balena-common/recipes-support/hostapp-update-hooks/files/hostapp-update-hooks

```diff
@@ -40,6 +42,15 @@ Options:
 EOF
 }
 
+DEBUG=$(jq -r '.debug // empty' "${CONFIG_PATH}")
```

Comment:
This undocumented `debug` `config.json` field could be re-used on other places where enabling tracing on demand is helpful.